inc/vr_update_renesas_gen3.hpp

```diff
@@ -10,12 +10,12 @@
 #define FW_WRITE             (0xE6)
 #define DMA_WRITE             0xC7
 #define DMA_READ             (0xC5)
-#define GEN3_CRC_ADDR        0x0094
+#define GEN3_CRC_ADDR        (0x00F8)
```

Comment:
Why these Addresses are getting changed? is it specific to version change ?

Reply:
These are the new registers to be accessed for the new parts in Venice